src/views/Pipeline.vue

```diff
@@ -176,6 +179,7 @@
           <JobConfiguration :title="title" :job="currentJob" :status="currentJobStatus" :type="freqType" :key="currentJob"/>
         </aside>
       </main>
+      <ion-loading v-if="isLoading" show-backdrop="true" spinner="crescent" />
```

Comment:
We will not use the ion-loading component, instead, we will use the predefined emitter to display the loader on click and dismiss the loader once the API call is a success or failure.

Reply:
Done Sir.